laceworksdk/api/container_registries.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+"""
+Lacework Container Registries API wrapper.
+"""
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class ContainerRegistriesAPI(object):
+    """
+    Lacework Container Registries API.
+    """
+
+    def __init__(self, session):
+        """
+        Initializes the ContainerRegistriesAPI object.
+
+        :param session: An instance of the HttpSession class
+
+        :return ContainerRegistriesAPI object.
+        """
+
+        super(ContainerRegistriesAPI, self).__init__()
+
+        self._session = session
+
+    def create(self,
+               name,
+               type,
+               enabled,
+               data,
+               org=False):
+        """
+        A method to create a new container registry.
+
+        :param name: A string representing the container registry name.
+        :param type: A string representing the container registry type.
+        :param enabled: An integer representing whether the container registry is enabled.
```

Comment:
nit. would it be better to use a python boolean here and just translate for data if it needs an int?